lib/elixir/test/elixir/string_test.exs

```diff
@@ -378,7 +378,7 @@ defmodule StringTest do
 
   test "normalize" do
     assert String.normalize("ŝ", :nfd) == "ŝ"
-    assert String.normalize("ḇravô", :nfd) == "ḇravô"
+    assert String.normalize("ḇravô", :nfd) == "ḇravô"
```

Comment:
I don't know if there is a way to view these differently in the console.  Visually I don't see anything different.  Its not until you look at the binary representation or the codepoints that an assessment can be made as by they are failing.   

``` elixir
test normalize (StringTest)
     lib/elixir/test/elixir/string_test.exs:379
     Assertion with == failed
     code: String.normalize("ḇravô", :nfd) == "ḇravô"
     lhs:  "ḇravô"
     rhs:  "ḇravô"
     diff: "ḇravôô"
     stacktrace:
       lib/elixir/test/elixir/string_test.exs:382: (test)
```

Console output
``` elixir
<<"ḇravô">> <> <<0>> = <<98, 204, 177, 114, 97, 118, 111, 204, 130, 0>>  # lhs
<<"ḇravô">> <> <<0>> = <<98, 204, 177, 114, 97, 118, 195, 180, 0>>  # rhs
```

This does make it difficult to diagnose a test failure.